ente photo fdroid build failed

```
2026-07-14T11:14:10.782543Z 01E 
2026-07-14T11:14:10.782655Z 01E FAILURE: Build failed with an exception.
2026-07-14T11:14:10.782744Z 01E 
2026-07-14T11:14:10.782790Z 01E * What went wrong:
2026-07-14T11:14:10.782847Z 01E Execution failed for task ':ente_qr:compileReleaseKotlin'.
2026-07-14T11:14:10.782897Z 01E > Error while evaluating property 'friendPathsSet$kotlin_gradle_plugin_common' of task ':ente_qr:compileReleaseKotlin'.
2026-07-14T11:14:10.782949Z 01E    > Could not resolve all files for configuration ':ente_qr:releaseCompileClasspath'.
2026-07-14T11:14:10.783004Z 01E       > Could not find com.github.markusfisch:zxing-cpp:v3.0.2.1.
2026-07-14T11:14:10.783065Z 01E         Required by:
2026-07-14T11:14:10.783096Z 01E             project :ente_qr
```

It seems that the jitpack repo should be added to https://github.com/ente/ente/blob/ca59cfd5388a65d5ae991a6817dbd0eb8e5cf0dd/mobile/packages/ente_qr/android/build.gradle#L18?

--- a/mobile/packages/ente_qr/android/build.gradle
+++ b/mobile/packages/ente_qr/android/build.gradle
@@ -19,6 +19,7 @@ rootProject.allprojects {
     repositories {
         google()
         mavenCentral()
+        maven { url 'https://jitpack.io' }
     }
 }
 

